manifests/uitpas/cid_logs.pp

```diff
@@ -0,0 +1,19 @@
+class profiles::uitpas::cid_logs (
+  Stdlib::Httpurl            $url             = 'http://cidlogs.publiq.be',
+  Stdlib::Ipv4               $service_address = '127.0.0.1',
+  Stdlib::Port::Unprivileged $service_port    = 8080,
+  String                     $data_dir        = '/data/cidlogs'
+) inherits ::profiles {
+
+  profiles::apache::vhost::reverse_proxy { $url:
+    destination => "http://${service_address}:${service_port}/"
+  }
+
+  file { $data_dir:
+    ensure => 'directory', 
+    owner  => 'logstash',
+    group  => 'logstash',
```

Comment:
Does the logstash package create these? If so, it's probably in a postinst script, and it will have a conditional statement around it to check if the user/group already exist. 
I usually (ab)use that to create the logstash user/group myself (with a fixed uid/gid) before installing the package.
Example here: https://github.com/cultuurnet/puppet-profiles/blob/main/manifests/aptly.pp#L18-L19 and https://github.com/cultuurnet/puppet-profiles/blob/main/manifests/users.pp#L3-L11
It is mostly useful for services that need storage, so we can assign an owner to the storage component without being dependent on the package being installed and to sync uids across servers.